src/main/groovy/util/ConfigHelper.groovy

```diff
@@ -10,16 +10,37 @@
 package util
 
 import grails.util.Holders
+import org.springframework.boot.info.GitProperties
 
 // See http://jira.codehaus.org/browse/GRAILS-6515
 class ConfigHelper {
 
+    /**
+     * Return the version of the application, as specified in build.gradle.
+     */
+    static String getAppVersion() {
+        // build.app.version is readable by war files, app.version by local builds
+        return Holders.grailsApplication.metadata.getProperty(
+            'build.app.version',
+            String,
+            Holders.grailsApplication.config.getProperty('app.version')
```

Comment:
My head is swimming in the details of what works and what has changed, but I'd like to follow conventions around the runtime and build properties as closely as possible.

I'm not sure if this is correct, but I saw application version retrieved in a more standardized way. Is there a reason we'd want to mess with that? I guess it's possible that we could set a different app version at build time, but do we actually do that?

If not then we should try to stick with the conventional
```
${g.meta(name: "info.app.version")}
```
and
```
Holder.grailsApplication.metadata.applicationVersion
```
https://docs.grails.org/3.3.10/guide/single.html#versioning

Also I think we can still use the g:meta tag lib, correct? I am cool with using the Holders.grailsApplication convention here since this is a utility, but in places where we can use taglibs (i.e. UI / controllers) I'd like to continue using the g:meta.

Reply:
Ah I see that is still the case. 

https://github.com/openboxes/openboxes/pull/3576/files#diff-598db67b4f4d6a15ec400ad2c40de3da3771b0976985c82f7df2ee1c5c7fd810R8

Reply:
Yeah, `<g.meta/>` "just works" for app version and grails version. git branch info is a little more complicated under bamboo, so I kept the helper function for that one.